app/staking/components/stakingTab/StakingTabs.tsx

```diff
@@ -0,0 +1,38 @@
+import styles from "./StakingTabs.module.scss";
+import Text from "@/components/text";
+
+interface StakingTabsProps {
+  handleTabChange: (txType: string) => void;
+  activeTab: string;
```

Comment:
maybe we can type this as:
`"delegate" | "undelegate" | "redelegate"`

we don't wan't weird bugs if something is misspelled. You can do an enum for this too. Up to you

Reply:
Yeah. Done this everywhere. However there isn't any chance of misspelled tx types as we're doing changing tx type when the tab changes.